src/screens/Learning.js

```diff
@@ -112,13 +115,15 @@ export default ({
             <ToolBar
               button={
                 <LanguageSwitcher
-                  firstLanguage={LANGUAGE_NAMES.EN}
-                  LeftText="EN"
-                  RightText="MA"
+                  firstLanguage={
+                    !secondLanguages ? LANGUAGE_NAMES.MG : LANGUAGE_NAMES.EN
+                  }
+                  LeftText={secondLanguages ? 'EN' : 'MA'}
+                  RightText={!secondLanguages ? 'EN' : 'MA'}
                   color="#FFFFFF"
                   iconType=""
                   iconName="swap-horiz"
-                  onPress={() => null}
+                  onPress={() => setSecondLanguages(!secondLanguages)}
```

Comment:
This should also dispatch action to change value in redux store so every component will have access to value of current language